<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPageLzy</class>
 <widget class="QWidget" name="SearchPageLzy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;查找用户和群聊&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="userScrollArea">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>229</width>
      <height>189</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>群聊：</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="groupScrollArea">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>229</width>
      <height>179</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="groupIdEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>群号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchGroupButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
